temp/harmonic_analysis.py

```diff
@@ -0,0 +1,106 @@
+"""Pseudocode for PyDPF-Post Harmonic Mechanical Simulations APIs."""
+
+import ansys.dpf.post as dpf
```

Comment:
@PProfizi what's the purpose of the temp folder? Is it pseudo code? I would keep it in a separate branch that we don't complete ever then.
(if not it might end up in master at some point...)